.github/workflows/dispatch-yocto-build-deploy.yml

```
@@ -0,0 +1,86 @@
+name: "Yocto"
+
+on:
+  workflow_dispatch:
+    # you may only define up to 10 `inputs` for a `workflow_dispatch` event
+    inputs:
+      # device-repo: # only required when testing from non-device repositories
+      #   description: balenaOS device repository (owner/repo)
+      #   required: true
+      #   type: string
+      #   default: balena-os/balena-generic
+      # device-repo-ref: # only required when testing from non-device repositories
+      #   description: balenaOS device repository tag, branch, or commit to build
+      #   required: false
+      #   type: string
+      #   default: master
+      # meta-balena-ref:
+      #   description: meta-balena ref if not the currently pinned version
+      #   required: false
+      #   type: string
+      yocto-scripts-ref:
+        description: balena-yocto-scripts ref if not the currently pinned version
+        required: false
+        type: string
+      machine:
+        description: yocto board name
+        required: true
+        type: string
+        default: generic-amd64
+      environment:
+        description: The GitHub Environment to use for the job(s) (production, staging, etc.)
+        required: true
+        type: choice
+        options:
+          - production
+          - staging
+      deploy-s3:
+        description: Whether to deploy images to S3
+        required: false
+        type: boolean
+        default: true
+      deploy-hostapp:
+        description: Whether to deploy a hostApp container image to a balena environment
+        required: false
+        type: boolean
+        default: true
+      deploy-ami: # TODO: can we get this from a source of truth like contracts?
+        description: Whether to deploy an AMI to AWS
+        required: false
+        type: boolean
+        default: false
+      sign-image: # TODO: can we get this from a source of truth like contracts?
```

```suggestion
      sign-image:
```
we can remove this comment, as Alex explained that image signing is a feature of the build, and not specific to the device-type

so keep the input as-is, but remove the comment